content/docs/install/completion.md

```diff
@@ -18,8 +18,8 @@ For example:
 $ dvc r # Press [tab] key
  -- dvc commands --
 remote  -- Manage remote storage configuration.
-remove  -- Remove outputs of DVC-file.
-repro   -- Check for changes and reproduce DVC-file and dependencies.
+remove  -- Remove outputs of `.dvc` file.
+repro   -- Check for changes and reproduce `.dvc` file and dependencies.
```

Comment:
Let's not change any instances in example command output like this. I extracted this task to the core repo: iterative/dvc/issues/3960

Reply:
But this one is wrong haha. `remove` is OK but `repro` is about `dvc.yaml`... Anyway, no action needed here since this file was removed from the PR.